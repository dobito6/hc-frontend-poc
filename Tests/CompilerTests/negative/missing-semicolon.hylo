//! no-std

fun f() {
  return return
}
