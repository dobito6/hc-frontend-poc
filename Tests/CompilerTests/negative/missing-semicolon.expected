negative/missing-semicolon.hylo:2.9: error: consecutive statements on the same line must be separated by ';'
  return return
        ^
